packages/devtools_app/lib/src/screens/memory/panes/leaks/instrumentation/model.dart

```diff
@@ -0,0 +1,44 @@
+// Copyright 2021 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:collection/collection.dart';
+
+enum LeakType {
+  notDisposed,
+  notGCed,
+  gcedLate,
+}
+
+LeakType _parseLeakType(String source) =>
```

Comment:
can we use enhanced enums to do this? If not we can create an extension method. See [example](https://github.com/flutter/devtools/blob/ecba4cd9dbdbe2e15ec0d299d24b8f23d9a151ab/packages/devtools_app/lib/src/screens/profiler/profile_granularity.dart#L29).

Reply:
I want the method to be private for the library, because it is not needed outside. If I make the method part of enum, it will become available for the library users.